src/test/java/uk/gov/companieshouse/api/strikeoffobjections/common/FormatUtilsUnitTest.java

```diff
@@ -1,4 +1,4 @@
-package common;
+package uk.gov.companieshouse.api.strikeoffobjections.common;
```

Comment:
You should now be able to remove the import statement below (for FormatUtils). This should be flagged by Sonar I'd have thought. Same in other classes.

Reply:
Wondering if our Sonar checks are working...

Reply:
Removed the imports and the public modifiers as sonar complains about them too. The local ide sonar checks don't seem to appear when you'd expect them it did not seem to complain about this at first. Perhaps it only responds to code changes.